MovieChecker/MovieChecker/BoxOfficeViewController.m

```diff
@@ -0,0 +1,148 @@
+//
+//  BoxOfficeViewController.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/21/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "BoxOfficeViewController.h"
+#import "BoxOfficeCell.h"
+#import "Movie.h"
+#import "RottenTomatoes.h"
+#import "MovieDetailViewController.h"
+// Define DLOG to log to NSLog when DEBUG is defined
+#ifdef DEBUG
+#define DLOG(...) NSLog(@"%s:%d %@", __PRETTY_FUNCTION__, __LINE__, [NSString stringWithFormat:__VA_ARGS__])
+#else
+#define DLOG(...) do {} while (NO)
+#endif
+
+@interface BoxOfficeViewController ()
+
+@property (nonatomic, strong) NSURLSession *session;
+
+@end
+
+@implementation BoxOfficeViewController
+
+- (id)initWithCoder:(NSCoder *)aDecoder
+{
+    self = [super initWithCoder:aDecoder];
+    if (self) {
+        NSURLSessionConfiguration *config = [NSURLSessionConfiguration ephemeralSessionConfiguration];
+        //[config setHTTPAdditionalHeaders:@{@"apikey": [RottenTomatoes apiKey]}];
+        self.session = [NSURLSession sessionWithConfiguration:config];
```

Comment:
It's preferable to directly set the ivar in an initializer method, rather than calling the setter (which is what happens with dot notation), but in this case, I'd think it might make more sense to move this setup to `viewDidLoad` and remove the initializer entirely.